docs/CoC.md

```diff
@@ -0,0 +1,211 @@
+list of community organizers is:
+
+* Sylvia van Os <sylvia@hackerchick.me> (lead organizer)
+* Koz Ross <koz.ross@retro-freedom.nz>
+
+When reporting, please include the following information, to the best of your
+knowledge and ability:
+
+* When and where the unacceptable behaviour occurred.
+* Who engages in unacceptable behaviour.
+* Which unacceptable behaviour(s) happened.
+* Any evidence for your report (such as screencaps, logs, etc)
+* Who (besides yourself) was affected by this behaviour.
+* Whether you want any additional actions taken by the community organizers to 
+  ensure that your safety and well-being remain intact.
+
+Additionally, community organizers are available to help community members
+engage with local law enforcement or otherwise help those experiencing
+unacceptable behaviour feel safe. In the context of in-person events, organizers
+will also provide escorts as desired by the person experiencing distress.
+
+## Addressing grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, you should notify the community organizers with a concise
+description of your grievance. You should include the following information:
+
+* What report of unacceptable behaviour you are responding to.
+* Why you believe specifically that you did not engage in this behaviour, or
+  why you believe the enforcement actions taken against you are
+  disproportionate.
+* Any evidence you can provide for your grievance.
+* What actions you wish to be taken to address your grievance if found genuine.
+
+If you have been temporarily, or permanently, banned without right of appeal,
+any grievances filed with community organizers regarding the incident which
+caused the ban will be ignored.
+
+## Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors;
+and other guests) to abide by this Code of Conduct in all 
+community venues, as well as in one-to-one communications pertaining to community 
+business. This Code of Conduct also applies to unacceptable behaviour occuring 
+outside the scope of community activities when such behaviour has the 
+potential to adversely affect the safety and well-being of community members.
+
+## Contact information
+
+This is the current list of community organizers, with appropriate contact
+information:
+
+* Sylvia van Os <sylvia@hackerchick.me> (lead organizer)
+* Koz Ross <koz.ross@retro-freedom.nz>
+
+## Modifications
+
+The community organizers reserve the right to modify this Code of Conduct to 
+take into account future situations that could not have been foreseen
+at the time of its writing, or to address lack of clarity or inconsistencies
+that become apparent with practice. Additionally, we must update this document
+with any changes in the community organizer team when appropriate.
+
+We will notify the community when any changes are made, along with a description
```

Comment:
Notify how?

Reply:
That's a very good question. Pext currently lacks a news 'blog', which would be the ideal place to have such notifications. Would this be something we could set up?

Reply:
Most people just state "changes may be made to these regulations without notice" making it the user's responsibility to check.

Reply:
@VenomousDingo I think that's extremely user-unfriendly. It's probably one of my _biggest_ pet peeves with such documents.

Reply:
My main issue with it is that it adds unnecessary extra work and responsibilities. Usually, participation means automatic agreement.

Reply:
I think Pext having a news 'blog' is probably something that should exist in any case. I don't expect that such changes are common - mostly it'd be changes in community org people - which in my view doesn't place unreasonable overheads to running such a thing. I'm willing to remove this section if @TheLastProject agrees with you, however.

Reply:
I agree that there should be an update/announcement blog, yes. Like a Twitter account that can post commits or even be used as a "support chat" of sorts if someone can't access IRC.

Reply:
I'm happy to operate such an account, on whatever service(s), if this is what we decide to do. However, I think this is orthogonal and requires its own issue.